docs/design/coreclr/profiling/davbr-blog-archive/Sample A Signature Blob Parser for your Profiler.md

```diff
@@ -3,42 +3,42 @@
 
 If your profiler plays with metadata, you've undoubtedly come across signature blobs. They’re used to encode type information for method definitions & references, local variables, and a whole lot more. They’re wonderfully compact, recursively versatile, and sometimes, well, challenging to parse. Fortunately, [Rico Mariani](https://docs.microsoft.com/en-us/archive/blogs/ricom/) was feeling generous one day, and churned out a simple parser that can read these types of signatures:
 
-MethodDefSig  
-MethodRefSig  
-StandAloneMethodSig  
-FieldSig  
-PropertySig  
+MethodDefSig
+MethodRefSig
+StandAloneMethodSig
+FieldSig
+PropertySig
 LocalVarSig
 
-Here are the files:  
-[sigparse.cpp](samples/sigparse.cpp) (Rico's signature parser)  
-[sigformat.cpp](samples/sigformat.cpp) (An example extension to the parser)  
+Here are the files:
+[sigparse.cpp](samples/sigparse.cpp) (Rico's signature parser)
+[sigformat.cpp](samples/sigformat.cpp) (An example extension to the parser)
 [PlugInToYourProfiler.cpp](samples/PlugInToYourProfiler.cpp) (Example code to plug the extension into your profiler)
```

Comment:
More suitable to be a list rather than explicit new line by either two spaces or a  `\`.

```suggestion
Here are the files:

- [sigparse.cpp](samples/sigparse.cpp) (Rico's signature parser)
- [sigformat.cpp](samples/sigformat.cpp) (An example extension to the parser)
- [PlugInToYourProfiler.cpp](samples/PlugInToYourProfiler.cpp) (Example code to plug the extension into your profiler)
```

Reply:
Agree